README.md

```diff
@@ -799,6 +803,78 @@ If you are using a feature flag with binary variants, the `StatusOverride` prope
 
 In the above example, the feature is enabled by the `AlwaysOn` filter. If the current user is in the calculated percentile range of 10 to 20, then the `On` variant is returned. Otherwise, the `Off` variant is returned and because `StatusOverride` is equal to `Disabled`, the feature will now be considered disabled.
 
+## Telemetry
+
+When a feature flag changed is deployed, it is often important to analyze its affect on an application. For example here are a few questions that may arise:
+
+* Are my flags enabled/disabled as expected?
+* Are targeted users getting access to a certain feature as expected?
+* What variant is a particular user seeing?
+
+These types of questions can be answered through the emission and analysis of feature flag evaluation events. This library supports emitting these events through telemetry publishers. One or many telemetry publishers can be registered to publish events whenever feature flags are evaluated.
+
+### Application Insights Telemetry Publisher
+
+This library provides a built-in telemetry publisher implementation that sends feature flag evaluation data to Application Insights. To take advantage of this, the `Microsoft.FeatureManagement.Telemetry.ApplicationInsights` package must be referenced. At that point, the Application Insights telemetry publisher can be registered.

```

Comment:
> **This library** provides a **built-in** telemetry publisher implementation...

The AppInsights publisher is released in a separate package, so it's not provided by "this library" or "built-in".

Reply:
I suppose the problem is that the term "this library" isn't concrete. I use the term to encompass all packages published from this repo. Not a specific package.

This sentence would be appropriate to say if it was also valid to say "This library provides a built-in mechanism to gate MVC controllers with a feature flag."

I can remove the "this library" terminology.

Reply:
As I go to rewrite this, I have a bit of an issue with the precedent here. This README is for every package in this repository. It's not just for the Microsoft.FeatureManagement package. It seems appropriate for this document to refer to this functionality as built-in for someone coming to review the functionality offered here.

Reply:
I agree with JImmy.

Reply:
Reworded to avoid the "this library" terminology, but kept the "built-in" terminology.

Also added a **Note:** to make it even more clear that this is not in the base package.